packages/controller/lib/setup/setupInstall.js

```diff
@@ -289,7 +289,11 @@ function Install(options) {
             console.error('Assuming that correct version is installed.');
         }
 
-        return this._npmInstall(npmUrl, options, debug);
+        try {
```

Comment:
catch from outside? return await is a bit strange imo ;-)

Reply:
but yes was this not so nice style before

Reply:
I think catching is the only reason to use return await. Catching from the outside means we have to catch and log in multiple locations though.

Reply:
I would also leave like this for now